cmd/encryption-v1.go

```diff
@@ -63,6 +65,8 @@ const (
 
 )
 
+type Context map[string]string
```

Comment:
What's the purpose of `Context`?

Reply:
removed this. Was used before rebasing on your PR for crypto (Un)/SealETag methods